core/modules/field_ui/field_ui.theme.inc

```diff
@@ -115,8 +115,11 @@ function theme_field_ui_view_modes($variables) {
 
   // Add the row for the default display mode.
   $row = array();
-  $row['label'] = $label . '<div class="description">' . $label_desc . '</div>';
-  $row['machine'] = t('default');
+  $row['label'] = theme('label_machine_name__node_display__' . backdrop_strtolower($element['#bundle']), array(
```

Comment:
why is the `strtolower` necessary here? Isn't `$element['#bundle']` the machine name of the bundle?

Reply:
Oh oh, never mind, this is just the default row. Marking as resolved!